server/monitor.go

```diff
@@ -51,10 +55,10 @@ func (m *Monitor) Monitor(hrt Heart, stop chan bool) error {
 
 // check attempts to beat a Heart several times within a timeout, returning the
 // log index at which the beat succeeded or an error
-func (m *Monitor) check(hrt Heart) (idx uint64, err error) {
+func check(hrt heart.Heart, ttl time.Duration) (idx uint64, err error) {
```

Comment:
Seems like an unnecessary coupling (I was also intending to have it replaceable for tests but didn't quite get that far)